themes/default/content/blog/pulumi-kubernetes-new-2022/index.md

```diff
@@ -0,0 +1,281 @@
+---
+date: "2022-10-23"
+title: "Pulumi+Kubernetes: Flux Integration, Inline Programs and SSA"
```

Comment:
is there room to fit in "New" or "Announcing" in here somehow? just to make it more clear that this is a launch.

Reply:
Tried to fit it in in latest update.